app/services/foreman/parameters/validator.rb

```diff
@@ -13,6 +13,10 @@ def validate!
           validate_regexp
         when "list", "array"
           validate_list
+        when "hash"
```

Comment:
ok, this bug is only reproducible in ruby 1.9.3, in which case this error message appears:
![screen shot 2015-11-10 at 1 02 58 pm](https://cloud.githubusercontent.com/assets/706359/11060832/667653ce-87ab-11e5-8d66-856b93c579c6.png)